src/test/java/org/opensearch/ad/rest/SecureADRestIT.java

```diff
@@ -65,47 +61,47 @@ public void setupSecureTests() throws IOException {
         aliceClient = new SecureRestClientBuilder(getClusterHosts().toArray(new HttpHost[0]), isHttps(), aliceUser, aliceUser)
             .setSocketTimeout(60000)
             .build();
-
+    
```

Comment:
Is this intended?

Reply:
Didn't change anything [here](https://github.com/opensearch-project/anomaly-detection/blob/feature/extensions/src/test/java/org/opensearch/ad/rest/SecureADRestIT.java#L68). 
I have removed the new line and pushed the change.